components/layout/hero/hero-video.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="hero-video__section section--bg-prim-blue">
+  <section class="hero-video">
+    <div class="hero-video__copy">
+      <h1 class="hero-video__title">We help people and organizations grow and
+        succeed.</h1>
+      <p class="hero-video__text">We’re at the intersection of people and tech.</p>
+    </div><!-- .hero-video__copy -->
+
+    <div class="hero-video__media">
+      <input
+        id="hero-video__controller"
+        class="hero-video__controller"
+        type="checkbox"
+      />
+      <label
```

Comment:
I think wrapping a video with a label is technically valid (videos are considered "phrasing content"), but something about it definitely doesn't feel great. If we can change this without breaking the interactivity elements, we should.